corn-maze.lic

```diff
@@ -0,0 +1,632 @@
+      DRC.message('The Script will STOW any loot, but to handle spillover consider adding other containers to your YAML as:')
+      DRC.message('cornmaze_containers:')
+      DRC.message('- backpack')
+      DRC.message('- duffel bag')
+      pause 2
+    end
+    
+    Flags.add('maze_done', 'leads you through the twisting passages and brings you to the exit')
+    Flags.add('Task_done', '10 of 10', '10 of the 10', '5 of the 5', '5 of 5', 'return to one of the Halflings and ASK HALFLING ABOUT TASK again')
+    @searching = false if args.restart
+    @looting = false if args.noloot
+    DRCI.stow_hands
+    find_start
+    get_task
+  end
+
+  def wait_for_command
+    reset_flags
+    Flags.reset('Task_done')
+    Flags.add('scarecrow', 'need to BUILD SCARECROW')
+    Flags.add('mice', 'need to WAVE to scare off the mice')
+    Flags.add('weeds', 'need to walk around the maze and PULL WEED')
+    Flags.add('traps', 'need to first SEARCH out a trap in each room')
+    Flags.add('grasshoppers', 'need to SEARCH in the rooms to find and catch the grasshoppers')
+    Flags.add('scream', 'need to move to different rooms in the maze and SCREAM')
+    Flags.add('landmarks', 'need to go around finding landmarks')
+    Flags.add('tokens', 'move around and SEARCH to find golden tokens')
+    Flags.add('corn', 'need to FORAGE CORN inside the maze')
+    Flags.add('halflings', 'need to go up to each Halfling and POKE them')
+    Flags.add('loot', 'You are not currently on a task in the Corn Maze')
+
+    fput "TASK"
+    pause 2
+    if Flags['scarecrow']
+      scarecrow_task
+    elsif Flags['mice']
+      mice_task
+    elsif Flags['weeds']
+      weeds_task
+    elsif Flags['traps']
+      traps_task
+    elsif Flags['grasshoppers']
+      grasshopper_task
+    elsif Flags['scream']
+      scream_task
+    elsif Flags['landmarks']
+      landmarks_task
+    elsif Flags['tokens']
+      tokens_task
+    elsif Flags['corn']
+      corn_task
+    elsif Flags['halflings']
+      halfling_task
+    elsif Flags['loot']
+      loot_task
+    else
+      echo "COMBAT TASK - CANCELING"
+    cancel_task
+    end
+  end
+
+  def cancel_task
+    DRC.bput 'TASK CANCEL', 'TASK CANCEL again in 30 seconds', 'Your current task has been canceled'
+    DRC.bput 'TASK CANCEL', 'TASK CANCEL again in 30 seconds', 'Your current task has been canceled'
+    pause 300
+    get_task
+  end
+
+  def traps_task
+    Flags.add('poisoned', 'You choke and gag on the poisonous gas')
+    @maze_path.each do |x|
+      exit if Flags['maze_done']
+      if !Flags['Task_done']
+        case DRC.bput('search', 'You don\'t think there are any traps here', 'tiny black dots are jumping around inside', 'You notice', 'already looked for traps around here')
+        when 'You notice'
+          waitrt?
+          fput 'disarm trap'
+          while checkstunned || checkwebbed
+            pause 1
+          end
+          waitrt?
+          while !checkstanding
```

Comment:
💡 `DRC.fix_standing`

Reply:
It's really not.  It's just the same thing in fput instead of bput, and it spams it needlessly.  I'll remove that.